lib/task-data/base-tasks/create-default-pollers.js

```diff
@@ -7,8 +7,7 @@ module.exports = {
     injectableName: 'Task.Base.Pollers.CreateDefault',
     runJob: 'Job.Pollers.CreateDefault',
     requiredOptions: [
-        'pollers',
-        'nodeId'
```

Comment:
When calling wsman-post-discovery taskgraph via workflow-tool, the node id is passed in as the first arg to the runGraph() function.  For some reason, the passed in node id is not found by the create-default-pollers task.  I've also tried passing node id in an options object, which also fails.